docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-OUTPUT.xml

```diff
@@ -473,17 +475,32 @@
               of the number of nodes on the target cluster(s).</entry>
             </row>
 
+            <row>
+              <entry><emphasis role="bold">PLANE</emphasis></entry>
+
+              <entry>Optional. Specifies writing the file to the specified
+              list of target planes. If omitted, the file is written to the
+              default plane for the cluster on which the workunit executes.
+              The number of physical file parts written to disk is always
+              determined by the number of nodes in the cluster on which the
+              workunit executes, regardless of the number of nodes on the
+              target. </entry>
+            </row>
+
             <row>
               <entry><emphasis>target</emphasis></entry>
 
               <entry>A comma-delimited list of string constants containing the
-              names of the clusters to write the file to. The names must be
-              listed as they appear on the ECL Watch Activity page or returned
-              by the Std.System.Thorlib.Group() function, optionally with
-              square brackets containing a comma-delimited list of
+              names of the clusters or planes to write the file to. The names
+              must be listed as they appear on the ECL Watch Activity page or
+              returned by the Std.System.Thorlib.Group() function, optionally
+              with square brackets containing a comma-delimited list of
```

Comment:
see prev. comments re. clarifying semantics between BM/Cluster and Containerized/Plane